packages/chart/src/ChartUtils.ts

```diff
@@ -1326,9 +1318,6 @@ class ChartUtils {
             axisProperty,
             figureAxisIndex
           );
-          if (layout[axisLayoutProperty] == null) {
-            layout[axisLayoutProperty] = this.makeLayoutAxis(axisType, theme);
-          }
```

Comment:
I wonder if this should set the axis to an empty object instead by default. Then the template part would still be applied by the template, but any other axis specifics (like range I think) should get applied to the axis. The code after calls `updateLayoutAxis` only if the axis exists. In subplots, the axis wouldn't exist (but it does seem to get created somewhere, I just don't know where)

```ts
layout: {
  xaxis: ...,
  xaxis2: { range: ... },
  template: {
    layout: {
      xaxis: { styleStuff }
    }
  }
}
```

Reply:
Good call. Doing this allowed me to restore most of the full tests I had deleted before which feels better to me.